scalableminds/webknossos
# fix missing bounding box regression

### Steps to test:
- Create a tracing, define a bounding box in the settings and check whether it is displayed.

### Issues:
- fixes https://discuss.webknossos.org/t/bounding-bbox-can-not-shown/426

------
